design-toolkit/components/src/components/dialog/styles.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { tv } from 'tailwind-variants';
+
+export const DialogStyles = tv({
+  slots: {
+    overlay: 'group/dialog absolute inset-0 flex items-center justify-center',
+    modal: [
+      'flex flex-col justify-center bg-surface-overlay align-start',
+      'group-size-small/dialog:rounded-medium',
+      'group-size-large/dialog:rounded-large',
+      'shadow-elevation-overlay',
+    ],
+    dialog: [
+      '[--dialog-size-small:280px]',
+      '[--dialog-size-medium:320px]',
```

Comment:
The types only support two sizes, are we intending to support this third size `medium`?